.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+
+Já leu? Então vamos às informações específicas deste repositório:
+- [Desafios](#desafios)
+- [Como configurar o ambiente de desenvolvimento](#como-configurar-o-ambiente-de-desenvolvimento)
+    - [Em Linux](#em-linux)
+    - [Em MacOS](#em-macos)
+    - [Em Windows](#em-windows)
+- [Formatação automática de código](#formação-automática-de-código)
+- [Mantendo](#mantendo)
+
+## Desafios
+O principal desafio deste repositório é ter cada vez mais raspadores de sites publicadores de diários oficiais, visando atingir os 5570 municípios brasileiros. Utilizamos o [Quadro de Expansão de Cidades](https://github.com/orgs/okfn-brasil/projects/12/views/13) para organizar o progresso do desafio. Consulte-o para localizar tarefas relevantes com as quais você pode contribuir. 
+
+Para te ajudar a desenvolver, utilize as orientações da página sobre [como escrever um novo raspador](https://docs.queridodiario.ok.org.br/pt/latest/escrevendo-um-novo-spider.html) disponível na [documentação técnica do Querido Diário](https://docs.queridodiario.ok.org.br/pt/latest/).
+
+## Como configurar o ambiente de desenvolvimento
+Os raspadores são desenvolvidos usando [Python](https://docs.python.org/3/) e o framework [Scrapy](https://scrapy.org). Você pode conferir [como instalar Python](https://www.python.org/downloads/) em seu sistema operacional. Com Python em seu computador, siga o passo-a-passo da configuração do ambiente de desenvolvimento:
+
+### Em Linux
+1. Faça um fork deste repositório e, com o terminal aberto em um diretório de preferência no seu computador, clone-o e acesse o novo diretório criado com o nome do repositório.
+``` console
+$ git clone <repositorio_fork>
+$ cd querido-diario
+```
+2. Crie um novo [ambiente virtual](https://docs.python.org/pt-br/3/library/venv.html) - que manterá as execuções do projeto isoladas de seu sistema.
+``` console
+$ python3 -m venv .venv
+```
+3. Ative o recém criado ambiente virtual
+``` console
+$ source .venv/bin/activate
+```
+4. Instale as [bibliotecas requeridas](querido-diario/data_collection/requirements-dev.txt).
+``` console
+$ pip install -r data_collection/requirements-dev.txt
+```
+5. Instale o pré-commit, uma ferramenta que, ao fazer o _commit_ do código, verifica se ele se adequa aos padrões do projeto.
+``` console
+$ pre-commit install
+```
+6. Seu ambiente de desenvolvimento está pronto! :tada:
+
+_Observação:_ Estas etapas precisam ser executadas apenas na primeira vez que interagir com o projeto durante a preparação do ambiente. Depois disso, basta ativar o ambiente virtual (passo 3) cada vez que for utilizar ou contribuir com o repositório.
+
+### Em MacOS
+[_em construção_]
+
+### Em Windows
```

Comment:
Acho que vale uma nota aqui afirmando que essa instalação só foi testada em Windows 10 e contribuições nesse tutorial são desejadas.

Reply:
Acrescentei a nota. Como o contributing ainda está pendente de melhorias (em com o que a pessoa pode contribuir no repositório) pensei em deixar para acrescentar a sugestão para contribuir com a documentação nessa seção quando ela for revisitada (guardei para não perder de vista).